ONSdigital/rm-sample-service
# Upgrade to java 17

# What and why?
Upgrade to java 17

# How to test?
deploy and ensure application starts up correctly tests still pass and sample still works correctly

# Trello
https://trello.com/c/4WlQT0gq/1626-upgrade-rm-sample-service-to-java-17
https://jira.ons.gov.uk/browse/RAS-557
